pkg/create/templates/test.go

```diff
@@ -43,7 +43,19 @@ container_exists() {
 }
 
 container_ip() {
-  docker inspect --format="{{"{{"}} .NetworkSettings.IPAddress {{"}}"}}" $(cat $cid_file)
+  if (echo "$OSTYPE" | egrep -qs 'darwin'); then
```

Comment:
IMHO we can use `grep` (or even `fgrep`) here instead of `egrep` because we don't use regular expressions here at all. Also I think that brackets could be omitted (`if echo "$OSTYPE" | grep -qs 'darwin'; then`).